### PR TITLE
Admin home page and logout  

### PR DESCRIPTION
**Pull request by HemaSAli**
Redirect to admin panel home page after login and adding logout function and check Authentication of login ..

### CODE REVIEW

File: public/css/mainStyle.css [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+}
+.logo {
+  padding-top: 0.5%;
+  padding-bottom: 0px;
+  height: 40px;
+}
+ul {
+  list-style-type: none;
+  list-style: none;
+  margin-top: 20%;
```
Comment (NouraldinS) [inline public/css/mainStyle.css]: Do not use relative margin and padding unless you're willing to fight for it.

File: src/controllers/index.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+const adminHomePage = require('./adminHomePage');
+const { authCheck } = require('./middleware');
 
 const router = express.Router();
 router.get('/admin/login', login.get);
 router.post('/admin/login', login.post);
-
-
+router.use(authCheck);
+router.get('/admin',adminHomePage.get);
```
Comment (NouraldinS) [inline src/controllers/index.js]: Fix indentation, and while you're doing it add a newline after requiring express, to separate module requires from file requires.

File: src/controllers/middleware.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const { parse } = require('cookie');
+const { verify } = require('jsonwebtoken');
+require('env2')('./config.env');
+
+exports.authCheck = (request, response, next) => {
```
Comment (NouraldinS) [inline src/controllers/middleware.js]: Fix indentation, and while you're here you could try adding a few comments to describe the function and the steps it takes.

File: src/views/layouts/adminLayout.hbs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (NouraldinS) [inline src/views/layouts/adminLayout.hbs]: Fix indentation.

File: src/views/partials/adminHead.hbs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Lato|Montserrat" rel="stylesheet">
+    {{#if styleFile}}
+    <link rel="stylesheet" href="/css/{{styleFile}}.css">
```
Comment (NouraldinS) [inline src/views/partials/adminHead.hbs]: Fix indentation.

File: src/views/partials/adminSidebar.hbs [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+      </a>
+      <a href="/admin/cohorts">
+        <li>
+          <img class="icon" src="/img/multiple-users-silhouette.png">
+          <span class="menu">Cohorts</span>
+        </li>
+      </a>
+
+      <a>
+        <li onclick="subMenu()">
```
Comment (NouraldinS) [inline src/views/partials/adminSidebar.hbs]: Fix indentation, and while you're here rename `subMenu` function to something more descriptive to the functionality.

File: src/views/partials/adminSidebar.hbs [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+      <ul id="sub" class="sub-menu">
+        <a href="Community.html">
+
+          <li>
+            <img class="icon" src="/img/collaboration.png">
+            <span class="menu">Community Project</span>
+          </li>
+        </a>
+
+        <a href="client.html">
```
Comment (NouraldinS) [inline src/views/partials/adminSidebar.hbs]: Do you have a route `client.html` handled?

**Review by NouraldinS — CHANGES_REQUESTED** [review]

File: src/views/partials/adminSidebar.hbs (hunk 6 above)
Comment (ali-7) [reply src/views/partials/adminSidebar.hbs]: @NoureldeanSaed it's for a sub menu, why change it ?

**Review by lubnaabd — APPROVED** [review]

**Review by ali-7 — APPROVED** [review]

**Review by NouraldinS — APPROVED** [review]

**Pull request merged**